src/main/data/98_job_limits.sql

```diff
@@ -0,0 +1,4 @@
+-- Populates the job_limits table.
+
+INSERT INTO job_limits (id, launcher, concurrent_jobs) VALUES
+    ('72DFEF9A-F6B1-482E-B2A0-16194247BE31', NULL, 8);
```

Comment:
The row with the `NULL` launcher contains the default limits.

Reply:
Yeah, it's either a username or, eventually a group name. There's a feature request to allow users to launch analyses on behalf of a group that they belong to.